lib/active_merchant/billing/gateways/blue_snap.rb

```diff
@@ -1,4 +1,5 @@
 require 'nokogiri'
+require 'pry'
```

Comment:
If I had a nickel for every time I've done this.

Speaking from experience, you might find it easier to do this inline when you add breakpoints, e.g. (`require 'pry'; binding.pry`), that way rubocop will prevent you from trying to check this `require` line into the code.